src/api/app/models/event/base.rb

```diff
@@ -151,13 +151,13 @@ def perform_create_jobs
       save
       Rails.logger.debug "PCJ #{inspect} #{create_jobs.inspect}"
       create_jobs.each do |job|
-        eclass = job.to_s.camelize.safe_constantize
-        raise "#{job.to_s.camelize} does not map to a constant" if eclass.nil?
-        djob = eclass.new(self)
-        raise("#{job.to_s.camelize} is not a CreateJob") unless djob.is_a?(CreateJob)
-        opts = {}
-        opts = { queue: eclass.job_queue } if eclass.methods.include? :job_queue
-        Delayed::Job.enqueue djob, opts
+        job_class = job.to_s.camelize.safe_constantize
```

Comment:
Will still need this method to schedule the jobs because we still rely on event.undone_jobs column.